web/controllers/homepage_controller.ex

```diff
@@ -1,15 +1,16 @@
 defmodule App.HomepageController do
   use App.Web, :controller
   import Ecto.Query, only: [from: 2, select: 3]
```

Comment:
When I compile the project I've got this warning: "Compiling 22 files (.ex) warning: unused import Ecto.Query.select/3web/controllers/homepage_controller.ex:3" However it seems that a "select" is used in get_content. I'm not sure if it is loaded automatically or if the select in get_content is different. Not a blocker